tests/cloud_functions/testing_utils.py

```
@@ -0,0 +1,224 @@
+        return self._subcollections[subcollection_name]
+
+
+class InMemoryFirestoreCollection:
+    """A class that simulates a Firestore collection in memory.
+
+    Attributes:
+        _docs (dict): A dictionary to store documents in the collection,
+            where keys are document IDs and values are InMemoryFirestoreDoc instances.
+    """
+
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)